controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { response } = require('../app')
+const {allTopicsData} = require('../models/topics.mode.controller')
+
+function getAllTopicsData(request, response, next) {
+    allTopicsData()
+    .then((topicsData) => {
+        response.status(200).send(topicsData)
+    })
+}
+
```

Comment:
If you're adding next make sure it's accompanied by a catch block with a next piece of error handling middleware, otherwise just leave as req and res until then.